src/App.tsx

```diff
@@ -758,31 +758,11 @@ function App(): ReactElement {
                     key: "plot",
```

Comment:
Should this whole Tabs section be a component? I'm debating on this because it feels like I'd be passing a lot of arguments down multiple levels in the hierarchy.

Reply:
If you think it would clean things up, but it doesn't seem like a high priority to me. I like it when an App file shows at a high level everything that is going to be rendered in the App, so having these two tabs here makes sense to me, whereas there are other items above this that are dealing with user interaction that could be factored out first I think